frontend/public/components/utils/categorize-catalog-items.tsx

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable no-undef */
```

Comment:
Shouldn't be a ts file if there are no components.

But this should move into the existing catalog component file since it's only used there.

Reply:
These are a set of methods dealing with categorization.  catalog-items.jsx is going to have a lot of code for filtering, vert. tabs, sorting, etc.. Trying to make code more modular